app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -270,75 +564,139 @@ class HomeActivityTest {
 
   @Test
   fun testHomeActivity_promotedCard_chapterNameIsCorrect() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_UPTIME_MILLIS)
+    storyProgressTestHelper.markRecentlyPlayedFractionsStory0Exp0(
+      profileId = profileId1,
+      timestampOlderThanOneWeek = false
+    )
     launch<HomeActivity>(createHomeActivityIntent(internalProfileId1)).use {
       testCoroutineDispatchers.runCurrent()
-      onView(
-        allOf(
-          withId(R.id.promoted_story_list_recycler_view),
-          withParent(
-            atPosition(R.id.home_recycler_view, 1)
-          )
-        )
-      ).check(matches(hasDescendant(withText(containsString("Prototype Exploration")))))
+      scrollToPosition(position = 1)
+      verifyTextOnHomeListItemAtPosition(
+        itemPosition = 1,
+        targetViewId = R.id.chapter_name_text_view,
+        stringToMatch = "What is a Fraction?"
+      )
     }
   }
 
   @Test
   fun testHomeActivity_promotedCard_storyNameIsCorrect() {
+    fakeOppiaClock.setFakeTimeMode(FakeOppiaClock.FakeTimeMode.MODE_UPTIME_MILLIS)
+    storyProgressTestHelper.markRecentlyPlayedFractionsStory0Exp0(
+      profileId = profileId1,
+      timestampOlderThanOneWeek = false
+    )
     launch<HomeActivity>(createHomeActivityIntent(internalProfileId1)).use {
       testCoroutineDispatchers.runCurrent()
       scrollToPosition(position = 1)
       verifyTextOnHomeListItemAtPosition(
         itemPosition = 1,
         targetViewId = R.id.story_name_text_view,
-        stringToMatch = "First Story"
+        stringToMatch = "Matthew Goes to the Bakery"
```

Comment:
Why is this different than the test above? They seem to be the exact same story 0/exp 0 being marked as recently played.

Reply:
Yeah, actually this was changed earlier. I changed to Fractions and so this tests from Fraction's story now. It doesn't make any differences though.